delfin/api/v1/alert.py

```diff
@@ -82,6 +83,15 @@ def _input_check(self, alert_source):
             user_name = alert_source.get('username')
             security_level = alert_source.get('security_level')
             engine_id = alert_source.get('engine_id')
+
+            # Validate engine_id, check octet string can be formed from it
+            try:
+                OctetString.fromHexString(engine_id)
+            except ValueError:
+                msg = "Invalid engine_id.It should be set of octets in " \
+                      "hexadecimal format."
```

Comment:
plz update the msg, such as " engine_id should be set of octets in hexadecimal format."